var/spack/repos/builtin/packages/libfirefly/package.py

```diff
@@ -0,0 +1,28 @@
+# Copyright 2013-2023 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class Libfirefly(CMakePackage):
+    """A standalone C++ Library for vectors calculations"""
+
+    homepage = "https://libfirefly.tbhaxor.com"
+    url = "https://github.com/tbhaxor/firefly/archive/refs/tags/v2.1.0.tar.gz"
+
+    maintainers("tbhaxor")
+
+    version("master", git="https://github.com/tbhaxor/firefly", branch="master")
```

Comment:
```suggestion
    url = "https://github.com/tbhaxor/firefly/archive/refs/tags/v2.1.0.tar.gz"
    git = "https://github.com/tbhaxor/firefly.git"

    maintainers("tbhaxor")

    version("master", branch="master")
```

@tbhaxor thanks for the pull request. Once quick suggestion, we can actually break the git url out into the main package attributes so that you won't have to redefine it if you decide to add other branches someday.